torch/_tensor_docs.py

```diff
@@ -1733,6 +1733,40 @@ def callable(a, b) -> number
 In-place version of :meth:`~Tensor.trunc`
 """)
 
+add_docstr_all('type',
+               r"""
+type(dtype=None, non_blocking=False, **kwargs) -> str or Tensor
+Returns the type if `dtype` is not provided, else casts this object to
+the specified type.
+
+If this is already of the correct type, no copy is performed and the
+original object is returned.
+
+Args:
+    dtype (type or string): The desired type
+    non_blocking (bool): If ``True``, and the source is in pinned memory
+        and destination is on the GPU or vice versa, the copy is performed
+        asynchronously with respect to the host. Otherwise, the argument
+        has no effect.
+    **kwargs: For compatibility, may contain the key ``async`` in place of
+        the ``non_blocking`` argument.
```

Comment:
It would be nice to mention that it's deprecated. Right now it's not clear form the docs